src/api/common/index.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   dropsToXrp: utils.dropsToXrp,
   xrpToDrops: utils.xrpToDrops,
   toRippledAmount: utils.toRippledAmount,
-  generateWallet: utils.generateWallet,
+  generateAddress: utils.generateAddress,
```

Comment:
There is not a 1 to 1 mapping of rippled functions to these JS routines but to stimulate ideas here are the names and return types we went with:
https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/SecretKey.h#L100